.github/workflows/release-please.yaml

```
@@ -0,0 +1,22 @@
+on:
+  push:
+    branches:
+      - "release/*"
```

Should we follow a simpler release model for this library?

We could start with a single main branch and add versions on top of it. If a need arise to maintain multiple release branches we can introduce back this logic.

```suggestion
    branches:
      - "main"
```